docs/code/IDataViewImplementation.md

```diff
@@ -447,29 +448,12 @@ do not, but the vast majority do.
 # Class Structuring
 
 The essential attendant classes of an `IDataView` are its schema, as returned
-through the `Schema` property, as well as the `IRowCursor` implementation(s),
+through the `Schema` property, as well as the `DataViewRowCursor` implementation(s),
 as returned through the `GetRowCursor` and `GetRowCursorSet` methods. The
 implementations for those two interfaces are typically nested within the
 `IDataView` implementation itself. The cursor implementation is almost always
 at the bottom of the data view class.
 
-# `IRow` and `ICursor` vs. `IRowCursor`
-
-We have `IRowCursor` which descends from both `IRow` and `ICursor`. Why do
-these other interfaces exist?
-
-Firstly, there are implementations of `IRow` or `ICursor` that are not
-`IRowCursor`s. We have occasionally found it useful to have something
-resembling a key-value store, but that is strongly, dynamically typed in some
-fashion. Why not simply represent this using the same idioms of `IDataView`?
-So we put them in an `IRow`. Similarly: we have several things that behave
-*like* cursors, but that are in no way *row* cursors.
-
-However, more than that, there are a number of utility functions where we want
-to operate over something like an `IRowCursor`, but we want to have some
-indication that this function will not move the cursor (in which case `IRow`
-is helpful), or that will not access any values (in which case `ICursor` is
-helpful).
```

Comment:
Since `ICursor ` no longer exist (and right now the only difference between `DataViewRow` and `DataViewRowCursor ` is very straightforward (i.e. that `DataViewRowCursor ` provides a `MoveNext()` method) I thought it would be redundant to keep this section.